deploy-odf/deploy.sh

```diff
@@ -162,6 +106,8 @@ if ! ./verify.sh; then
             exit 1
         fi
     done
+
+    sleep 30
```

Comment:
sleep 30 or wait for resources?

Reply:
Can't recall but I believe this was here already and I just moved it. 

I think we can remove it but I would probably do it in a separate PR